### PR TITLE
Add art-template@4.9.1 w/ npm auto-update

### LINKED ISSUE #11276: [Request] Add art-template
**Library name:** art-template
**Git repository url:** https://github.com/aui/art-template
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** MIT
**Official homepage:** https://aui.github.io/art-template/
**Wanna say something? Leave message here:** 

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/45346261-request-add-art-template?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11276, cc @aui

Pull request for issue: #11276 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/aui/art-template
 * Official website (optional, not the repository): https://aui.github.io/art-template/
 * NPM package url (optional): https://www.npmjs.com/package/art-template
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 427
   - Count of stars: 4,944
   - Count of forks: 1,740
 * NPM download stats (optional):
   - Downloads in the last day: 166
   - Downloads in the last week: 726
   - Downloads in the last month: 5,196

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/art-template/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "art-template",
+  "version": "4.9.1",
+  "filename": "template-web.js",
+  "description": "JavaScript template engine",
+  "homepage": "https://aui.github.io/art-template/",
+  "keywords": [
+    "template",
+    "engine",
+    "art"
```
Comment (sufuf3) [inline ajax/libs/art-template/package.json]: I think if add the followings keywords can let it better.
```js
    "util",
    "functional",
```
ref: https://github.com/aui/art-template/blob/3.0.1/package.json#L6-L7

**Comment by dakshshah96** [comment]
@sufuf3 Done.

**Comment by sufuf3** [comment]
@dakshshah96 it's great!
if commit message `Add v4.9.0 of art-template manually, cc #11276` can change to `Add v4.9.0 of art-template manually from GitHub, cc #11276` and `Add v2.0.2~v3.1.3 of art-template manually, cc #11276` change to `Add v2.0.2~v3.1.3 of art-template manually from npm, cc #11276`
It will be useful to us to trace the commit message.

Thank you for your contribution.

**Comment by dakshshah96** [comment]
@sufuf3 Done 👍

**Comment by dakshshah96** [comment]
@extend1994 @hare1039 @clairetsai818 Can you help review this PR please?

**Comment by dakshshah96** [comment]
@extend1994 Isn't `Add v2.0.2~v3.1.3 of art-template manually from npm, cc #11276` more readable than `Add old versions of art-template manually, cc #11276` since we have two commits for adding old versions from npm and GitHub respectively.

**Comment by extend1994** [comment]
@dakshshah96 Sorry for that I did not say it in a clear way. The original one (which you used) is too many chars for reading on the GitHub. You can take a look at https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project :smh

**Comment by extend1994** [comment]
Sorry!! I accidentally press the button and reopen it now.

**Comment by dakshshah96** [comment]
@extend1994 No problems!

**Comment by sufuf3** [comment]
@dakshshah96 Let's modify commit message from `Add v2.0.2~v3.1.3 of art-template manually from npm, cc #11276` to `Add old versions of art-template manually, cc #11276`, what do you think?
Because the commit message short than 50 chars is helpful to us to trace git log, as https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project said.

After the commit message is fine, I think this PR will merge soon.

Thank you for your contribution.

**Comment by sufuf3** [comment]
@cdnjs/library-reviewer 
Please help review this PR, Thank you.

**Comment by extend1994** [comment]
@cdnjs/intern3 Please help review this PR because there are more commits here (compared to others).

**Comment by mercyg** [comment]
Do we need to squash the commit?

**Comment by PeterDaveHello** [comment]
@mercyg Nope, as each of them has their own meaning :) Good question!

**Comment by sufuf3** [comment]
@dakshshah96 
thanks for contribution.
Would you mind working on this and helping improve this PR as the [comment](https://github.com/cdnjs/cdnjs/pull/11285#pullrequestreview-63640247)?
If you have any question, please let us know.

Many thanks.

**Comment by jiang539** [comment]
i can't found art-template at https://cdnjs.com/

**Comment by extend1994** [comment]
ping @dakshshah96 
@jiang539 The library can't be found because the PR is not merged yet.

**Comment by jiang539** [comment]
Looking forward to
@extend1994

**Comment by extend1994** [comment]
@PeterDaveHello Can we take over this PR? It's high priority library.

**Comment by sashberd** [comment]
@sufuf3 @extend1994 v4.8.0 was added from git. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Pull request closed without merging**